test/frame.jest.js

```diff
@@ -265,4 +265,20 @@ describe('Frame Management', function() {
     expect(frame2.isDetached()).toBe(false);
     expect(frame1).not.toBe(frame2);
   });
+  it.fail(FFOX)('should refuse to display x-frame-options:deny iframe', async({page, server}) => {
+    server.setRoute('/x-frame-options-deny.html', async (req, res) => {
+      res.setHeader('Content-Type', 'text/html');
+      res.setHeader('X-Frame-Options', 'DENY');
+      res.end(`<!DOCTYPE html><html><head><title>login</title></head><body style="background-color: red;"><p>dangerous login page</p></body></html>`);
+    });
+    await page.goto(server.EMPTY_PAGE);
+    const refusalText = new Promise(f => {
+      page.on('console', msg => {
+        if (msg.text().match(/Refused to display/i))
```

Comment:
Chrome sometime logs an error about the favicon missing, so we filter the error messages.